users/functions/index.js

```diff
@@ -24,6 +24,18 @@ exports.deleteWarrantUser = functions.auth.user().onDelete((user) => {
       .catch((error) => console.log(error));
 });
 
+// Use this instead of `onCreate` when you want to set claims on sign in
```

Comment:
Do you mean sign up?

Reply:
No, I meant you should use that function if you wanted to use it in conjunction with the `setUserClaims` function before sign in but let me try to make it clearer